src/util.ts

```diff
@@ -416,3 +421,21 @@ export const movingAveDataSort = (
   }
   return aveArray
 }
+
+export const sevenDayDataMerge = (data: Data[]): DataPlusSevenDayAve[] => {
+  const sevenDayDataArr: DataPlusSevenDayAve[] = []
+  if (Array.isArray(data) === false) return sevenDayDataArr
+  const sevenDayData = movingAveDataSort(data)
+  if (sevenDayData.length === 0) return sevenDayDataArr
```

Comment:
See if you can combine these `if` statements. Perhaps move line 428 up, and then combine the `if`s on lines 427 and 429 using an `||` operand.

Reply:
Made changes according to comment.